README.md

```diff
@@ -24,16 +24,21 @@ You will need the following:
 
 ### What's Working? ###
 Most features of the game are available -- and we're at a state where you can complete the full campaign without issue.
+##### Original Features #####
 - Single player modes (Campaigns, custom levels, custom campaigns)
 - All diseases, objects, rooms are available (see below section for outstanding anomalies)
 - All events (emergencies, earthquakes, epidemics, VIP visits)
 - Micromanagement
+- Music/Jukebox and gameplay videos (see below section for oustanding anomalies)
+- Cheats (naughty!)
+##### New Features #####
 - Full HD resolution and beyond!
```

Comment:
Our hidpi support is kinda poor, best to leave this at just widescreen or full HD support?